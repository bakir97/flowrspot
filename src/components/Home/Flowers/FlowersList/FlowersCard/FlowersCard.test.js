import React from "react";
import { shallow } from "enzyme";
import FLowersCard from "./FLowersCard";
describe("FLowersCard", () => {
  it("should render correctly", () => {
    const component = shallow(<FLowersCard flower={{}} />);
    expect(component).toMatchSnapshot();
  });
  it("should render star", () => {
    const component = shallow(<FLowersCard user flower={{}} />);
    expect(component).toMatchSnapshot();
  });
});
