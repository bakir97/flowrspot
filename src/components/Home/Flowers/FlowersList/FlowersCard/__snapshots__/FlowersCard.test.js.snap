// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FLowersCard should render correctly 1`] = `
<div
  className="card"
>
  <Content />
  <img
    className="image"
  />
</div>
`;

exports[`FLowersCard should render star 1`] = `
<div
  className="card"
>
  <Star
    user={true}
  />
  <Content />
  <img
    className="image"
  />
</div>
`;
